contactaccessrights/js/src/access-rights/models/location.js

```diff
@@ -0,0 +1,14 @@
+define([
+  'access-rights/modules/models',
+  'access-rights/services/location',
+  'common/services/api',
```

Comment:
Why not first define the ones we're going to use a reference to, then the ones we don't need a reference to?